python/Gaffer/Application.py

```diff
@@ -86,6 +93,10 @@ def root( self ) :
 	## Called to run the application and return a status value.
 	def run( self ) :
 
+		if self.parameters()["help"].getValue() :
+			os.system( "gaffer -help " + self.path )
```

Comment:
Yeah, once I found the already working way, I wasn't sure I needed the second way... but I'd already gone to such lengths to implement it that I wasn't ready to back off. Maybe I'll ask around and see what some others think...